apps/frontend/src/components/AccountSidebar.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Box,
+  Divider,
+  Drawer,
+  List,
+  ListItem,
+  Typography,
+} from "@mui/material";
+import { useLocation, useNavigate } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { useStyles } from "../styles/pageStyles";
+import AccountSidebarItems from "./AccountSidebarItems";
+
+const AccountSidebar = () => {
```

Comment:
If this component took a selectedItem prop, you could avoid dealing with the useEffect hook to find the selected  item from the path. Then AccountSidebarItems (should be renamed to singular AccountSidebarItem) can just take a "selected" prop instead of selectedItem and setSelectedItem

Example:
`const AccountSidebar = ({selectedItem}: string)`
...
`<AccountSidebarItem ...otherProps selected={selectedItem === "My Profile"}`

AccountSidebarItem.tsx:
`<ListItemButton onClick={navigate(path)}...  backgroundColor: selected ? style : transparent`>